Source/cmake/OptionsWPE.cmake

```
@@ -110,6 +110,13 @@ WEBKIT_OPTION_DEPEND(ENABLE_DOCUMENTATION ENABLE_INTROSPECTION)
 
 WEBKIT_OPTION_CONFLICT(USE_SOUP2 ENABLE_WPE_1_1_API)
 
+# Supported platforms.
+WEBKIT_OPTION_DEFINE(USE_WPEWEBKIT_PLATFORM_WESTEROS "Whether to enable support for the Westeros platform" PUBLIC OFF)
+WEBKIT_OPTION_DEFINE(USE_WPEWEBKIT_PLATFORM_BCM_NEXUS "Whether to enable support for the Broadcom Nexus platform" PUBLIC OFF)
+WEBKIT_OPTION_DEFINE(USE_WPEWEBKIT_PLATFORM_INTEL_CE "Whether to enable support for the Intel CE" PUBLIC OFF)
+WEBKIT_OPTION_DEFINE(USE_WPEWEBKIT_PLATFORM_RPI "Whether to enable support for the raspberry pi" PUBLIC OFF)
```

Nit: capitalization:

```suggestion
WEBKIT_OPTION_DEFINE(USE_WPEWEBKIT_PLATFORM_RPI "Whether to enable support for the Raspberry Pi" PUBLIC OFF)
```